### PR TITLE
Forms: WGAG 2.1 AA Assessment

### PR DESCRIPTION
**Pull request by BrahimMahadi**
### This pull request includes the WCAG 2.1 & 2.2 Accessibility assessment done for the forms page

General checklist
- [x] Updated forms documentation to include the accessibility assessment reports
- [x] Page was tested against WCAG for accessibility
- [x] Documentation is bilingual

### CODE REVIEW

File: common/scaffolding/reports/a11y-2.json [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/scaffolding/forms.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of the forms page whose purpose is to test all native forms related elements, if they are aligned with our design and are compliant to our accessibility guideline when used as is without any special customization.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ""
```
Comment (duboisp) [inline common/scaffolding/reports/a11y-2.json:16]: Missing the CSS selector that identify what that has been tested for WCAG. Is it the whole page?

**Review by duboisp — CHANGES_REQUESTED** [review]
I will wait for your report @AtlakM , we will add it to this PR. And can you fix the missing CSS selector

**Comment by AtlakM** [comment]
> I will wait for your report @AtlakM , we will add it to this PR. And can you fix the missing CSS selector

https://github.com/AtlakM/GCWeb/commit/e77b916064763d4f26a4e58de13f45ea7bfad209

File: common/scaffolding/reports/a11y-3.json [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+  }
+}
```
Comment (duboisp) [inline common/scaffolding/reports/a11y-3.json:631]:
```suggestion
      "earl:outcome": "earl:passed"
    }
  ]
}
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Just a few little change

**Review by duboisp — APPROVED** [review]
Reviewed and it's look good. 

This is only a content change

File: common/scaffolding/reports/a11y-3.json [hunk 3]
```diff
@@ -0,0 +1,625 @@
...
+        }
+      ],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The landmark error result is a false positive so it passes the axe test.",
+      "earl:outcome": "earl:passed",
+    }
```
Comment (duboisp) [inline common/scaffolding/reports/a11y-3.json:623]: The comma need to be removed.

```suggestion
      "earl:outcome": "earl:passed"
    }
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]
Looks good, the validation error was fixed.

**Pull request merged**